package.json

```diff
@@ -124,6 +124,12 @@
           "description": "Preserve empty content/whitespace in a tag.",
           "scope": "window"
         },
+        "xml.format.preserveNewlines": {
+          "type": "boolean",
```

Comment:
maybe we should just authorize int values instead:  [0 ,1, 2, 3]. Set default to 2, 0 means no line preserved